.env.example

```diff
@@ -1,95 +0,0 @@
-### Documentation available at https://docs.wodby.com/stacks/php/local
```

Comment:
I checked and I have my env.example file on my local, it was not deleted.
![env](https://user-images.githubusercontent.com/30270640/77103201-553d5c00-6a1a-11ea-826e-0774dbcb14f4.png)

Reply:
Something's not right for sure because it is clearly missing in your branch: https://github.com/Pronovix/food/tree/issue/7903

What does `git status` say?

Reply:
True. I noticed that it is a hidden file, however even after removing it from .gitignore, it's till not visible for git, I can't add and push it.

Reply:
Solved. I forced git to add the file